developer/yk-deploy.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/sh
+
+# Create a local K8S cluster (using Kind) with a Yunikorn scheduler
+
+arch=arm64         # either amd64 or arm64
+ns=yunikorn
+registry=gresearch
+
+# The base directory which contains all the Yunikorn repo dirs below
+yk_repos_base=$HOME/src/yunikorn
```

Comment:
Suggestion:
 
- Create two Make targets for both architectures.
- Make `arch` configurable by passing the arch argument to the script and pass this argument through a make target.
```shell
arch = $1
```
and the make targets can be something like
```shell
deploy-arm64:
    ./developer/yk-deploy.sh "arm64"
deploy-amd64:
    ./developer/yk-deploy.sh "amd64"
```